pkgs/development/python-modules/pymupdf/nix-support.patch

```diff
@@ -0,0 +1,17 @@
+--- a/setup.py
++++ b/setup.py
+@@ -36,10 +36,14 @@ LIBRARIES = {
+     "opensuse": OPENSUSE,
+     "fedora": FEDORA,
+     "alpine": ALPINE,
++    "nix": FEDORA,

```

Comment:
Not sure if this is the right thing to do.

Reply:
Ah, base the [patch](https://github.com/pymupdf/PyMuPDF/commit/d9b2d42019e5705a1c6621ea0cdfa26da1ce9ad5), I found the FEDORA dependencies same with nix environment, so I directly use it, this will be remove on next release.